packages/boot/test/orchestration/contract-upgrade.test.ts

```diff
@@ -70,23 +80,21 @@ test('resume', async t => {
 
   // XXX golden test
   const getLogged = () =>
-    JSON.parse(storage.data.get('published.sendAnywhere.log')!).values;
-
-  // This log shows the flow started, but didn't get past the name lookup
-  t.deepEqual(getLogged(), ['sending {0} from cosmoshub to cosmos1whatever']);
-
-  t.log('upgrade sendAnywhere with fix');
-  await evalProposal(
-    buildProposal('@agoric/builders/scripts/testing/fix-buggy-sendAnywhere.js'),
-  );
+    JSON.parse(storage.data.get('published.send-anywhere.log')!).values;
 
+  // This log shows the flow started, but didn't get past the IBC Transfer settlement
   t.deepEqual(getLogged(), [
     'sending {0} from cosmoshub to cosmos1whatever',
     'got info for denoms: ibc/FE98AAD68F02F03565E9FA39A5E627946699B2B07115889ED812D8BA639576A9, ibc/toyatom, ibc/toyusdc, ubld, uist',
     'got info for chain: cosmoshub cosmoshub-4',
     'completed transfer to localAccount',
   ]);
 
+  t.log('upgrade sendAnywhere with fix');
```

Comment:
you changed the name of `start-buggy-sendAnywhere` to `init-send-anywhere`, which I take to mean it's no longer buggy. So there's no fix. So this new proposal isn't really upgrading in a meaningful way, is it? Just restarting?

Reply:
I renamed this log to `null upgrading sendAnywhere`